e2e/playwright.config.ts

```diff
@@ -7,7 +7,7 @@ const config: PlaywrightTestConfig = {
     retries: process.env.CI ? 2 : 1,
     workers: 4,
     fullyParallel: true,
-    timeout: 90 * 1000,
+    timeout: 150 * 1000,
```

Comment:
I'd prefer if we used a per-test timeout override (`test.setTimeout(150000)`) rather than doing this globally as most tests shouldn't require all that time.

Reply:
Right, good point.